openhantek/CMakeLists.txt

```diff
@@ -29,8 +29,17 @@ add_subdirectory(translations)
 
 add_definitions(-DVERSION="${CPACK_PACKAGE_VERSION}")
 
+# if windows and mingw then set WIN32 flag to remove cmd window
+# and it's good place to add mac appbundle later
```

Comment:
Please remove those "TODO" style comments. You can add them to the commit message instead.